dialect_postgres.go

```diff
@@ -1179,31 +1146,40 @@ func parseOpts(name string, o values) {
 	for _, p := range ps {
 		kv := strings.Split(p, "=")
 		if len(kv) < 2 {
-			errorf("invalid option: %q", p)
+			return fmt.Errorf("invalid option: %q", p)
 		}
 		o.Set(kv[0], kv[1])
 	}
+
+	return nil
 }
 
 func (p *pqDriver) Parse(driverName, dataSourceName string) (*core.Uri, error) {
 	db := &core.Uri{DbType: core.POSTGRES}
-	o := make(values)
-	var err error
+	var (
+		dbName string
```

Comment:
It seems local variable `dbName` is unnecessary.

Reply:
You mean it should be inlined and db.DbName should be set directly on line 1165 and 1176?

Reply:
Why is that a bug? It is not an unused variable.

Reply:
@daniellee yes, I think so.
This PR will fix a bug issue so that I labeled it `bug`.